src/collections/Breadcrumb/Breadcrumb.js

```diff
@@ -0,0 +1,106 @@
+import _ from 'lodash'
+import cx from 'classnames'
+import React, { PropTypes } from 'react'
+import META from 'src/utils/Meta'
+import { customPropTypes } from '../../utils/propUtils'
+import * as sui from '../../utils/semanticUtils'
+import BreadcrumbDivider from './BreadcrumbDivider'
+import BreadcrumbSection from './BreadcrumbSection'
+
+/**
+ * A breadcrumb is used to show hierarchy between content.
+ */
+function Breadcrumb(props) {
+  const {
+    children, divider, icon, size, sections, ...rest,
+  } = props
+
+  const classes = cx(
+    'ui',
+    size,
+    'breadcrumb'
+  )
+
+  if (!sections) {
+    return <div className={classes} {...rest}>{children}</div>
+  }
+
+  let dividerJSX = <Breadcrumb.Divider />
+
+  if (divider) {
+    dividerJSX = <Breadcrumb.Divider>{divider}</Breadcrumb.Divider>
+  }
+
+  if (icon) {
+    dividerJSX = <Breadcrumb.Divider icon={icon} />
+  }
```

Comment:
Can we use `else` chaining here to prevent extra function calls and conditional checks?:

```jsx
let dividerJSX

if (divider) {
  dividerJSX = <Breadcrumb.Divider>{divider}</Breadcrumb.Divider>
} else if (icon) {
  dividerJSX = <Breadcrumb.Divider icon={icon} />
} else {
  dividerJSX = <Breadcrumb.Divider />
}
```